package.json

```diff
@@ -29,12 +29,12 @@
 		"lint-php:fix": "composer run-script lint-fix",
 		"lint-php:skip-warnings": "composer run-script lint:skip-warnings",
 		"pretty": "prettier --write \"assets/prototype/**/*.js\"",
-		"prewatch": "npm run check-engines",
 		"test": "npm run lint && npm run test-js",
 		"test-php": "cd tests && phpunit",
 		"test-js": "node scripts/test.js",
 		"test-js:coverage": "npm run test-js -- --coverage",
-		"test-js:coverage-ci": "npm run test-js -- --coverage --maxWorkers 1"
+		"test-js:coverage-ci": "npm run test-js -- --coverage --maxWorkers 1",
+		"watch": "node scripts/watch.js"
```

Comment:
We might not need `dev` command or we might replace that one with this, not sure.